model/relabel/relabel_test.go

```diff
@@ -857,25 +880,6 @@ func TestConfig_UnmarshalThenMarshal(t *testing.T) {
 		name      string
 		inputYaml string
 	}{
-		{
-			name: "Values provided",
-			inputYaml: `source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_port]
-separator: ;
-regex: \\d+
-target_label: __meta_kubernetes_pod_container_port_number
-replacement: $1
-action: replace
-`,
-		},
-		{
-			name: "No regex provided",
-			inputYaml: `source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_port]
-separator: ;
-target_label: __meta_kubernetes_pod_container_port_number
-replacement: $1
-action: keepequal
-`,
-		},
```

Comment:
Why were these lines deleted?

Reply:
sorry it was for testing purposes, git add *

Reply:
Recommend you always look over what is changed in the PR, at the time you create it.
Every change should be covered by the description, either top-level or in individual commits.

Reply:
thanks